src/test/app/NFToken_test.cpp

```diff
@@ -6841,12 +7156,15 @@ class NFToken0_test : public beast::unit_test::suite
         static FeatureBitset const all{supported_amendments()};
         static FeatureBitset const fixNFTDir{fixNFTokenDirV1};
 
-        static std::array<FeatureBitset, 5> const feats{
-            all - fixNFTDir - fixNonFungibleTokensV1_2 - fixNFTokenRemint,
+        static std::array<FeatureBitset, 6> const feats{
+            all - fixNFTDir - fixNonFungibleTokensV1_2 - fixNFTokenRemint -
```

Comment:
We verify the behavior of unit tests with and without new amendments because we...

1. Don't know when (or if) the amendment will be approved.  We don't want to damage existing behavior while adding an amendment.
2. We like for the current version of rippled to process ledgers that are at least a couple of years old.  This is very helpful for ledger replay.  So we want to verify that the unit tests show old behavior when recent amendments are not enabled.

Hopefully that explanation makes some sense.

Reply:
thanks for the explanation! I forgot about the amendment voting process :) 👍